autospec/infile_update_spec.py

```diff
@@ -80,6 +87,7 @@ def write_cmd_files(bb_dict, dirname):
             with open(filename, 'a') as cmdfp:
                 cmdfp.write("# Infile parser added the following lines:\n")
                 cmdfp.write("\n".join(cmd) + "\n")
+            print_infile("File: {} changed".format(cmd_mappings.get(cmd_name)))
```

Comment:
Could this say something like `"Suggestions added to {} file"` to make it a little more intuitive for the user?